maintainers/maintainer-list.nix

```diff
@@ -12845,5 +12845,11 @@
     github = "mbprtpmnr";
     githubId = 88109321;
   };
+  dillenburg = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.